v2/datastream-to-spanner/src/test/resources/DataStreamToSpanner100GbLT/spanner-schema-interleaving.sql

```diff
@@ -10,7 +10,7 @@ CREATE TABLE IF NOT EXISTS person2 (
      FK_ID2 INT64,
      ID INT64 NOT NULL,
 ) PRIMARY KEY(ID, FK_ID2),
-  INTERLEAVE IN PARENT person1 ON DELETE CASCADE;
+  INTERLEAVE IN person1;
```

Comment:
Is there a separate test which preserves the FKA - `ON DELETE CASCADE`? Can you point me to it?

Reply:
We will only do LTs for Interleave IN as discussed. There is a FK LT but not with ON DELETE CASCADE: https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/v2/datastream-to-spanner/src/test/resources/DataStreamToSpanner100GbLT/spanner-schema-foreign-key.sql